README.md

```diff
@@ -1 +1,28 @@
-# github-profile-fetcher
\ No newline at end of file
+# Github Profile Fetcher
+---
+## How to run the App!
+- Clone this repo down to your machine
+```
+git clone https://github.com/partyzl/github-profile-fetcher
```

Comment:
tried doing this but because your changes have reached main yet I just keep pulling an empty readme. im just too thick to realise what command I should be using to pull your actual project from staging

Reply:
I did it, I just used `git clone --single-branch --branch staging git@github.com:partyzl/github-profile-fetcher.git`

Reply:
git checkout staging